### DISCUSSION TITLE
Heimdall update results in error

### DISCUSSION THREAD

**Discussion by elmakus** [tteck/Proxmox#128] [Q&A]
after update from your script service is not starting

systemctl status heimdall
* heimdall.service - Heimdall
     Loaded: loaded (/etc/systemd/system/heimdall.service; enabled; vendor preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Tue 2022-04-05 22:09:57 UTC; 4s ago
    Process: 428 ExecStart=/usr/bin/php artisan serve --port 7990 --host 0.0.0.0 (code=exited, status=1/FAILURE)
   Main PID: 428 (code=exited, status=1/FAILURE)
        CPU: 94ms

Apr 05 22:09:57 heimdall systemd[1]: heimdall.service: Main process exited, code=exited, status=1/FAILURE
Apr 05 22:09:57 heimdall systemd[1]: heimdall.service: Failed with result 'exit-code'.

**Comment by tteck** [comment]
updating from which version?

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/120
